packages/web-components/src/components/va-button/va-button.tsx

```diff
@@ -50,6 +53,22 @@ export class VaButton {
    */
   @Prop({ reflect: true }) disabled?: boolean = false;
 
+  /**
+   * If `true`, the click event will not fire.
```

Comment:
Is this description accurate for the loading prop?

Reply:
Yes, this is handled the same way as the disabled prop by `preventDefault()` and `preventPropogation()` on line 168.

Reply:
oh ok yes that makes sense. I was expecting it to mention that it will display the loading indicator too.

Reply:
oh, I should add that! Thanks!